### PR TITLE
Added option to keep IBAN number instead or returning old account numbers

### PR DESCRIPTION
**Pull request by bravoman**

### CODE REVIEW

File: src/Kingsquare/parser/banking/mt940/engine.php [hunk 1]
```diff
@@ -152,7 +152,7 @@ protected function parseStatementAccount() {
 		}
 
 		// SEPA / IBAN
-		if (preg_match('/:25:[A-Z0-9]{8}([\d\.]+)*/', $this->getCurrentStatementData(), $results)
+		if (preg_match('/:25:([A-Z0-9]{8}[\d\.]+)*/', $this->getCurrentStatementData(), $results)
```
Comment (bravoman) [inline src/Kingsquare/parser/banking/mt940/engine.php]: Account is sanitized anyway.

File: src/Kingsquare/parser/banking/mt940/engine.php [hunk 2]
```diff
@@ -326,10 +326,13 @@ protected function sanitizeAccount($string) {
 			' ' => '',
 			'GIRO' => 'P',
 		);
-		// crude IBAN to 'old' converter
-		if (preg_match('#[A-Z]{2}[0-9]{2}[A-Z]{4}(.*)#', $string, $results) && !empty($results[1])) {
-			$string = $results[1];
-		}
+        
+        if(\Kingsquare\Parser\Banking\Mt940::$removeIBAN) {
```
Comment (bravoman) [inline src/Kingsquare/parser/banking/mt940/engine.php]: Condition removing of IBAN

File: src/Kingsquare/parser/banking/mt940.php [hunk 3]
```diff
@@ -8,9 +8,12 @@
  * @license http://opensource.org/licenses/MIT MIT
  */
 class Mt940 extends Banking {
-	/* @var Banking\Mt940\Engine engine */
+	/** @var Banking\Mt940\Engine */
 	protected $engine;
 
+    /** @var bool */
+    public static $removeIBAN = true; // defaults to true for BC
+
```
Comment (bravoman) [inline src/Kingsquare/parser/banking/mt940.php]: Static for now, because otherwise the engine needed the same config.

**Comment by fruitl00p** [comment]
Looks good... Although i'd like to see another way of passing in the config, this will suffice for the time being!

Thanks for the update!

**Pull request merged**